### PR TITLE
Fix typos to Docker 20.10 alert

### PR DESCRIPTION
**Pull request by AlecRust**

### CODE REVIEW

File: alerts/docker_2010.markdown [hunk 1]
```diff
@@ -9,18 +9,26 @@ homeassistant: ">0.7"
...
 
-There are 2 parts to this issue. The first is the removal of filtering of images over the API, this results in images not beeing recognized by the Supervisor correctly.
+There are 2 parts to this issue. The first is the removal of filtering of images over the API, this results in images not being recognized by the Supervisor correctly.
 
 The second issue is changes in how the systemd service for Docker works, which results in the service `hassio-supervisor` not being able to start on system boot.
 
 ## Workaround
 
-The only known workaround at this point is downgrading Docker to the previous version. The examples given below is for Debian 10. If that is not your OS, check the documentation for your OS's package manager on how to perform the downgrade.
+The only known workaround at this point is downgrading Docker to the previous version. The examples below are for Debian 10 and Ubuntu 20. If that is not your OS, check the documentation for your OS's package manager on how to perform the downgrade.
```
Comment (frenck) [inline alerts/docker_2010.markdown:18]: Ubuntu is not a supported system and should be removed from that.

```suggestion
The only known workaround at this point is downgrading Docker to the previous version. The examples below are for Debian 10. If that is not your OS, check the documentation for your OS's package manager on how to perform the downgrade.
```

File: alerts/docker_2010.markdown [hunk 2]
```diff
@@ -9,18 +9,26 @@ homeassistant: ">0.7"
...
 
+**Debian 10**
```
Comment (frenck) [inline alerts/docker_2010.markdown:20]:
```suggestion
```

File: alerts/docker_2010.markdown [hunk 3]
```diff
@@ -9,18 +9,26 @@ homeassistant: ">0.7"
...
 apt install containerd.io=1.3.9-1
 ```
 
+**Ubuntu 20**
+```bash
+apt install docker-ce=5:19.03.14~3-0~ubuntu-focal
+apt install docker-ce-cli=5:19.03.14~3-0~ubuntu-focal
+apt install containerd.io=1.3.9-1
+```
+
```
Comment (frenck) [inline alerts/docker_2010.markdown:33]: Ubuntu is not supported by the Home Assistant project.
```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: alerts/docker_2010.markdown (hunk 3 above)
Comment (AlecRust) [reply alerts/docker_2010.markdown:32]: Just thought there would be quite a few people looking for these commands like I was, but fair enough.

File: alerts/docker_2010.markdown [hunk 4]
```diff
@@ -9,13 +9,13 @@ homeassistant: ">0.7"
...
 
-There are 2 parts to this issue. The first is the removal of filtering of images over the API, this results in images not beeing recognized by the Supervisor correctly.
+There are 2 parts to this issue. The first is the removal of filtering of images over the API, this results in images not being recognized by the Supervisor correctly.
 
 The second issue is changes in how the systemd service for Docker works, which results in the service `hassio-supervisor` not being able to start on system boot.
 
 ## Workaround
 
-The only known workaround at this point is downgrading Docker to the previous version. The examples given below is for Debian 10. If that is not your OS, check the documentation for your OS's package manager on how to perform the downgrade.
+The only known workaround at this point is downgrading Docker to the previous version. The examples below are for Debian 10. If that is not your OS, check the documentation for your OS's package manager on how to perform the downgrade.
```
Comment (frenck) [inline alerts/docker_2010.markdown:18]:
```suggestion
The only known workaround at this point is downgrading Docker to the previous version. The example below is for Debian 10. If that is not your OS, check the documentation for your OS's package manager on how to perform the downgrade.
```

**Review by frenck — APPROVED** [review]

**Pull request merged**